core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -91,21 +91,24 @@ public AlluxioMasterRestServiceHandler(@Context ServletContext context) {
 
   /**
    * @summary get the configuration map, the keys are ordered alphabetically.
+   * @param raw whether the unresolved, raw property values should be returned, null means false
    * @return the response object
    */
   @GET
   @Path(GET_CONFIGURATION)
   @ReturnType("java.util.SortedMap<java.lang.String, java.lang.String>")
-  public Response getConfiguration() {
+  public Response getConfiguration(@QueryParam("raw") final Boolean raw) {
```

Comment:
is this required to be `Boolean` rather than `boolean`? I am asking because in that case of `boolean`, you don't need to handle `null`